src/Display/Tooltip/Tooltip.tsx

```diff
@@ -22,9 +22,9 @@ const Tooltip: React.FunctionComponent<Props> = ({
 
   const hideTooltipIfTouchOutside = React.useCallback(
     (event: TouchEvent | React.TouchEvent) => {
-      const hasTouchedOutsideOfTooltipContent = !contentRef.current.contains(
-        event.target as HTMLElement
-      );
+      const hasTouchedOutsideOfTooltipContent =
+        contentRef.current &&
```

Comment:
Should we also check isShow here?

Reply:
Since `hideTooltipIfTouchOutside` is wrapped in `useCallback` already, I would not like to add `isShow` in dependency that will recreate  `hideTooltipIfTouchOutside` function when `isShow` is updated.

Reply:
Oh I didn't see the useCallback, I thought it's a function. Yes, makes sense else it would be called every time when isShow updates.